dwave/samplers/sa/src/cpu_sa.h

```diff
@@ -31,6 +31,9 @@
     #endif
 #endif
 
+typedef enum {GIBBS, METROPOLIS} mcmc_t;
+typedef enum {SEQUENTIAL, RANDOM} varorder_t;
```

Comment:
```suggestion
enum Proposal {
    Gibbs,       ///< Short description here
    Metropolis,  ///< Short description here
};

enum VariableOrder {
    Sequential,   ///< Short description here
    Random,       ///< Short description here
};
```

We use Google's C++ Style guide, see their guidance on [enumerator naming](https://google.github.io/styleguide/cppguide.html#Enumerator_Names).

Reply:
I'm adding k per the style guide.

Reply:
We have note used `k` historically. So let's not start now (I know I am contradicting myself).